package.json

```diff
@@ -56,8 +57,11 @@
     "deploy-styleguide": "gatsby build --prefix-paths && gh-pages -d public",
     "storybook": "start-storybook -s ./dist -p 6006",
     "build-storybook": "build-storybook",
-    "build": "webpack -p --config ./webpack/webpack.config.js --display-error-details",
-    "watch": "webpack -d --watch --config ./webpack/webpack.config.js --display-error-details",
-    "develop": "concurrently --raw \"yarn watch\" \"yarn storybook\""
+    "babel": "npx babel components -w -d dist/js --ignore 'components/**/*.component.js','components/**/*.stories.js' --presets=@babel/preset-env,babel-preset-minify --plugins=babel-plugin-drupal-behaviors",
+    "build-babel": "npx babel components -d dist/js --ignore 'components/**/*.component.js','components/**/*.stories.js' --presets=@babel/preset-env,babel-preset-minify --plugins=babel-plugin-drupal-behaviors",
+    "webpack": "webpack -d --watch --config ./webpack/webpack.config.js --display-error-details",
+    "build-webpack": "webpack -p --config ./webpack/webpack.config.js --display-error-details",
+    "postinstall": "yarn build-webpack",
+    "develop": "concurrently --raw \"yarn webpack\" \"yarn babel\" \"yarn storybook\""
```

Comment:
I'd like to have documentation about these somewhere. e.g. "Here are the commands you'll use, and when/why to use them."

Reply:
Yea, we definitely need that cause this is pretty involved.